src/test/java/org/codehaus/plexus/util/xml/pull/MXSerializerTest.java

```diff
@@ -55,4 +56,19 @@ private String expectedOutput() {
         out.append("</root>");
         return out.toString();
     }
+
+    /**
+     * Tests MJAVADOC-793.
+     */
+    @Test
+    public void testMJAVADOC793() throws IOException {
```

Comment:
This should be renamed. This is not related to the Javadoc plugin. IT simply does not allow null values. `testNullValues()`?

Reply:
Done: `testWriteNullValues()`.